src/modals.less

```diff
@@ -15,24 +19,24 @@
 }
```

Comment:
Hey, I realized that this piece of code is redundant and not being used anywhere which I totally forgot to remove before pushing it. Can you please remove this?
```
 .transition(@transition) {
   -webkit-transition: @transition;
   transition: @transition;
}
```

Reply:
I'll do it ASAP

Reply:
Thanks 👍

Reply:
And it's done!

Reply:
Good catch!